addons/mail/static/src/discuss/call/public_web/discuss_sidebar_call_participants.js

```diff
@@ -59,12 +59,9 @@ export class DiscussSidebarCallParticipants extends Component {
 
     get sessions() {
         const sessions = [...this.props.thread.rtcSessions];
-        return sessions.sort((s1, s2) => {
-            return (
-                s1.channelMember?.persona?.nameOrDisplayName?.localeCompare(
-                    s2.channelMember?.persona?.nameOrDisplayName
-                ) ?? 1
-            );
-        });
+        return sessions.sort(
+            (s1, s2) =>
+                s1.channelMember?.persona?.name?.localeCompare(s2.channelMember?.persona?.name) ?? 1
```

Comment:
The default fallback to 1 looks very wrong too. The sort return should be deterministic and symmetrical, it can't return 1 when checking a vs b or b vs a.
It should at least compare ids as last ressort.

Reply:
Could we have a UI test to cover the issues?